src/components/DetailArticle.jsx

```diff
@@ -1,25 +1,47 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useContext } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import {
   fetchArticleById,
   fetchComments,
+  fetchDeleteComment,
   fetchVotesUpdate,
 } from "../Utils/api";
 import { FaArrowAltCircleUp, FaArrowAltCircleDown } from "react-icons/fa";
 
 import "./DetailArticle.css";
 import { AddComment } from "./AddComment";
+import UserContext from "../contexts/UserContext";
 
 export const DetailArticle = () => {
   const [comments, setComments] = useState([]);
   const [article, setArticle] = useState(null);
   const [votes, setVotes] = useState(0);
   const [error, setError] = useState(null);
   const [loading, setLoading] = useState(true);
+  const [deletingComment, setDeletingComment] = useState(false);
+  const { currentUser } = useContext(UserContext);
 
+  
   const { article_id } = useParams();
   const navigate = useNavigate();
 
+
+  const handleDeleteComment = (comment_id) => {
+  console.log(comment_id)
+    fetchDeleteComment(comment_id)
+      .then(() => {
+        setLoading(false);
+        window.location.reload()
```

Comment:
Using `window.location.reload()` to reload everything defeats the point of using state, i.e. using state we can just reload the component that needs to be reloaded rather than reloading the whole page. In this case, only the list of comments would need to be re-rendered.
Please remove `window.location.reload()` and instead update the state of the `commentList` so it gets rid of the relevant comment.